Muehle_GUI.ipynb

```diff
@@ -497,30 +497,6 @@
     "            board[2].fill_arc(BOARD_SIZE*positions[ring][cell][0], BOARD_SIZE*positions[ring][cell][1], PIECE_RADIUS, 0, 360)"
    ]
   },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "`winner_popup()` erzeigt ein PopUp-Fenster mit Glückwünschen für den übergebenen Spieler. Die Funktion wird aufgerufen, wenn das Spiel zu Ende ist."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "def winner_popup(winner):\n",
-    "    msg = \"Herzlichen Glückwunsch! \\nSpieler \" + str(winner) + \", du hast gewonnen!\"\n",
-    "    popup = tk.Tk()\n",
-    "    popup.wm_title(\"Gewonnen!\")\n",
-    "    label = ttk.Label(popup, text=msg, font=(\"Verdana\", 10))\n",
-    "    label.pack(side=\"top\", fill=\"x\", pady=10)\n",
-    "    B1 = ttk.Button(popup, text=\"Yeah!\", command = popup.destroy)\n",
-    "    B1.pack()\n",
-    "    popup.mainloop()"
-   ]
-  },
```

Comment:
Müssten wir das dann nicht in der Statuszeile ausgeben?
""Herzlichen Glückwunsch! \\nSpieler \" + str(winner) + \", du hast gewonnen!\"\n""

Reply:
Ja genau. Dafür gibt's ja schon ein Issue (#51)

Reply:
akzeptiert